### ISSUE TITLE
Installation problem

### ISSUE THREAD

**Issue by abretin** [swina/IceMood#8]
After installing all the requirements, to the login page, when I enter the access (user icemood and password: icemood) nothing happens. No action happens.

**Comment by swina** [comment]
Hi, 
I am not working on this project anymore but I will try to help you in order to solve your problem.
Authentication is very basic for this release. Authentication data are stored in the application/config/config.php file
```
//LOGIN
$config['admin'] = 'icemood';
$config['admin_email'] = 'your email';
$config['admin_pw'] = 'icemood';

```
You can find the login function in application/models/Admin_model.php
```
public function validate(){
        // grab user input
        $username = $this->input->post('user_email');
        $password = $this->input->post('user_pass');
		if ( $username == trim($this->config->item ( 'admin')) ){
			if ( $password == trim($this->config->item ( 'admin_pw')) ) {
				$data = array(
                    'adminid' => 1,
                    'aname' => $username,
                    'aemail' => $this->config->item ( 'admin_email' ),                   
                    'validated' => true
                    );
				$this->session->set_userdata($data);
                return "1";
			} 
		} 
		
    }
```

As you can see it's very basic authentication and you can set your authentication as per your needs.
Did you check from the console for any error?

You should also create the folloing folders in the root folder where you installed icemood:

```
audio
    |___ mp3
    |___ playlists
    |___ trash
```

Try to put an mp3 file in the mp3 folder

Another possible problem could be the database setting. Icemood doesn't use any database, but in this release there is database configuration file in application/config/database.php.


Try to set a valid database connection (like a fake db with no data).
Did you install locally or on a server. Do you have a link where I can check?

I hope this will help. Please contact me for any further problem you will find.

Have fun

**Comment by swina** [comment]
Hi I am working on a more stable release to correct some issues and to fix login problems.

**Comment by abretin** [comment]
Yes, I have a link where you can check. I tried to watch but I really do not see the problem.
it's here : http://podcast.moveupradio.fr/

**Comment by abretin** [comment]
Hi, 

I found this problem on apache logs.
http://i.imgur.com/5n95Ex8.png
How can I solve it?

**Comment by fconejerose** [comment]
Hi, help please....
  Im facing the same problem.
  Click on Login does anything.
  In /var/log/http/radiorocknation.cl-error_log showing:

[Tue Jun 06 10:39:36 2017] [error] [client 192.168.1.1] File does not exist: /var/www/html/radiorocknation.cl/admin/ajax, referer: http://www.radiorocknation.cl/admin/
[Tue Jun 06 10:39:37 2017] [error] [client 192.168.1.1] File does not exist: /var/www/html/radiorocknation.cl/admin/ajax, referer: http://www.radiorocknation.cl/admin/

can you help me, please
Thank you very much
